README.md

```diff
@@ -37,6 +37,11 @@ gcb-visualizer --output my-pipeline.jpg <your-cloudbuild-yaml>
 ```
 The current supported output formats are jpg, jpeg, dot and png.
 
+## Example Output
+Given an example input of something like the [Google Cloud Go Builder cloudbuild.yaml](https://github.com/GoogleCloudPlatform/cloud-builders/blob/master/go/cloudbuild.yaml)
+you should get an output that looks something like this 
+![Go Builder output](./example/example.png)
```

Comment:
This image is not the expected one. The expected one should be a straight line flowchart.